Classes/Bookmark/BookmarkCell.swift

```diff
@@ -84,14 +84,14 @@ final class BookmarkCell: SwipeSelectableCell {
                 string: "\(viewModel.bookmark.owner)/\(viewModel.bookmark.name)",
                 attributes: [
                     .font: Styles.Text.secondaryBold.preferredFont,
-                    .foregroundColor: Styles.Colors.Gray.light.color,
+                    .foregroundColor: Styles.Colors.Gray.light.color
```

Comment:
Just some general PR feedback: making all these small format changes alongside an actual change make the entire PR very noisy, and makes it more difficult for the reviewer to find actual changes for the PR. If you want to make style changes, I'd recommend making those separate PRs!

Reply:
Agree, although at least it seems to be [a separate commit](55955be).